packages/tools/client-debugger/client-debugger/src/test/ClientUtilities.ts

```diff
@@ -0,0 +1,127 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { ConnectionState } from "@fluidframework/container-loader";
+import { ContainerSchema, FluidContainer, IFluidContainer } from "@fluidframework/fluid-static";
+import {
+	ITinyliciousAudience,
+	TinyliciousClient,
+	TinyliciousContainerServices,
+} from "@fluidframework/tinylicious-client";
+
+import { initializeFluidClientDebugger as initializeFluidClientDebuggerBase } from "@fluid-tools/client-debugger";
+
+/**
+ * This module contains Fluid Client utilities, including Container creation / loading.
+ */
+
+/**
+ * Type returned from when creating / loading the Container.
+ */
+export interface ContainerLoadResult {
+	container: IFluidContainer;
+	services: TinyliciousContainerServices;
+}
+
+/**
+ * Basic information about the container, as well as the associated audience.
+ */
+export interface ContainerInfo {
+	containerId: string;
+	container: IFluidContainer;
+	audience: ITinyliciousAudience;
+}
+
+/**
+ * Creates a new Fluid Container from the provided client and container schema.
+ *
+ * @param client - The Tinylicious service client.
+ * @param containerSchema - Schema with which to create the container.
+ *
+ * @throws If container creation or attaching fails for any reason.
+ */
+export async function createFluidContainer(
+	client: TinyliciousClient,
+	containerSchema: ContainerSchema,
+): Promise<ContainerInfo> {
+	// Create the container
+	let createContainerResult: ContainerLoadResult;
+	try {
+		createContainerResult = await client.createContainer(containerSchema);
+	} catch (error) {
+		console.error(`Encountered error creating Fluid container: "${error}".`);
+		throw error;
+	}
+
+	const { container, services } = createContainerResult;
+
+	// Attach container
+	let containerId: string;
+	try {
+		containerId = await container.attach();
+	} catch (error) {
+		console.error(`Encountered error attaching Fluid container: "${error}".`);
+		throw error;
+	}
+
+	return {
+		container,
+		containerId,
+		audience: services.audience,
+	};
+}
+
+/**
+ * Loads an existing Container for the given ID.
+ *
+ * @throws If no container exists with the specified ID, or if loading / connecting fails for any reason.
+ */
+export async function loadExistingFluidContainer(
```

Comment:
I don't think tests in this package will need this code path. This is really only for our sandbox app in the case where the user navigates to the URL with the container ID already specified. Our tests will likely always need to generate new containers.

Reply:
Make sense! Removing this function.